ServerNG.md

```diff
@@ -0,0 +1,155 @@
+Infinispan ServerNG
+====================
+
+Infinispan ServerNG is a reboot of Infinispan's server which addresses the following:
+
+* small codebase with as little duplication of already existing functionality (e.g. configuration)
+* embeddable: the server should allow for easy testability in both single and clustered configurations
+* RESTful admin capabilities
+* Logging using [JBoss Logging logmanager](https://github.com/jboss-logging/jboss-logmanager)
+* Security using [Elytron](https://github.com/wildfly-security/wildfly-elytron)
+
+# Layout
+
+The server is laid out as follows:
+
+* `/bin` scripts
+   * `server.sh` server startup shell script for Unix/Linux
+   * `server.ps1` server startup script for Windows Powershell 
+* `/lib` server jars
+   * `infinispan-server.jar` uber-jar of all dependencies required to run the server.
```

Comment:
Is this only one jar with all the needed classes, or just a jar to reference all the dependencies?
In any case this would make any patching/updates worse as the users want to know what the difference is.
In that case there is no simple way to replace a module or 3rd party like JGroups

Reply:
Right, a single jar doesn't provide any advantage.
Patching, however, in the world of docker images, becomes less of an issue, as the user will just pull a more recent image and won't see what is inside it anyway.